theia-go-docker/Dockerfile

```diff
@@ -24,9 +24,15 @@ WORKDIR /home/theia
 ADD $version.package.json ./package.json
 RUN yarn --cache-folder ./ycache && rm -rf ./ycache
 RUN yarn theia build
+# See : https://github.com/theia-ide/theia-apps/issues/34
+RUN adduser --disabled-password --gecos '' theia
+RUN chmod g+rw /home && \
+    mkdir -p /home/project && \
+    chown -R theia:theia /home/project;
 EXPOSE 3000
 ENV SHELL /bin/bash
 RUN mkdir /root/.go
 RUN echo '{"toolsGopath":"/root/go"}' > /root/.go/go.json
```

Comment:
We should use the ```theia``` user's home instead

Reply:
Did you understand I meant this line: 
``` RUN echo '{"toolsGopath":"/root/go"}' > /root/.go/go.json```?

Reply:
Forgive me, I probably misunderstood, would fix shortly